crypto/keyring/armor.go

```diff
@@ -153,7 +151,7 @@ func encryptPrivKey(privKey cryptotypes.PrivKey, passphrase string) (saltBytes [
 	}
 
 	key = crypto.Sha256(key) // get 32 bytes
-	privKeyBytes := legacy.Cdc.Amino.MustMarshalBinaryBare(privKey)
+	privKeyBytes := CryptoCdc.MustMarshalBinaryBare(privKey)
```

Comment:
could we maybe keep the file in the crypto directory and use `keyring.CryptoCdc`? or would that generate dependency cycles?

Reply:
moving the file is also ok but I think it's out of scope from the fix

Reply:
Moving the file is to avoid import cycles.
keyring import armor,
the codec used by Armor changed from `legacy.Cdc.Amino` to `keyring.CryptoCdc`.